src/SpectatorView.Native/SpectatorView.Compositor/Compositor/DeckLinkDevice.cpp

```diff
@@ -286,6 +288,12 @@ class OutputScheduler : public IDeckLinkVideoOutputCallback
 
     }
 
+    void SetLatencyPreference(float latencyPreference)
+    {
+        minFramesQueued = 1 + latencyPreference * (minFramesQueued - 1);
```

Comment:
>minFramesQueued [](start = 51, length = 15)

Should this be defaultMinFramesQueued?  Otherwise it looks like each call to SetLatencyPreference will keep reducing the minFramesQueued without ever growing the value

Reply:
Thinking more, maybe actually I meant MAX_NUM_OUTPUT_FRAMES?

---
In reply to: [432713687](https://github.com/microsoft/MixedReality-SpectatorView/pull/381#discussion_r432713687) [](ancestors = 432713687)

Reply:
good catch!
`defaultMinFramesQueued` had a couple of frames left for mitigating delays by the decklink output, so using `MAX_NUM_OUTPUT_FRAMES` and a `latencyPreference` of `1` would disable this.